--- origsrc/pypebbles-0.1/pypebbles.py	2004-03-23 16:27:12.000000000 -0600
+++ src/pypebbles-0.1/pypebbles.py	2007-06-01 00:56:11.125000000 -0500
@@ -22,9 +22,10 @@
 import pprint, profile, pstats
 from pygame.locals import *
 
+path = '/usr/share/pypebbles'
 
 def load_image(name, colorkey=None):
-    fullname = os.path.join('data', name)
+    fullname = os.path.join(path, 'data', name)
     try:
         image = pygame.image.load(fullname)
     except pygame.error, message:
